src/components/Sum.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => (
+  // eslint-disable-next-line react/react-in-jsx-scope
```

Comment:
You should add React import in this file and delete this line

Reply:
Fixed